code/parseParams.m

```diff
@@ -13,17 +13,19 @@
 def_date = date;
 %
 expected_family = {'gaussian','poisson'};
-validScalar = @(x) isnumeric(x) && isscalar(x) && (x >= 0);
+% Changed the conditions to accommodate command line calls to GLG_Instance.
```

Comment:
Inputs will be strings when coming from the command line.  Need to support both numeric and string inputs.